src/main/java/org/opentripplanner/gtfs/mapping/TransitModeMapper.java

```diff
@@ -51,33 +51,24 @@ public static TransitMode mapMode(int routeType) {
       return null;
     } else if (routeType >= 1600 && routeType < 1700) { //Self drive
       return TransitMode.BUS;
-    } else if (routeType >= 1700 && routeType < 1800) { //Miscellaneous Service
+      // 1700 is mapped to CARPOOL further down
+    } else if (routeType >= 1701 && routeType < 1800) { //Miscellaneous Service
       return null;
     }
     /* Original GTFS route types. Should these be checked before TPEG types? */
-    switch (routeType) {
-      case 0:
-        return TransitMode.TRAM;
-      case 1:
-        return TransitMode.SUBWAY;
-      case 2:
-        return TransitMode.RAIL;
-      case 3:
-        return TransitMode.BUS;
-      case 4:
-        return TransitMode.FERRY;
-      case 5:
-        return TransitMode.CABLE_CAR;
-      case 6:
-        return TransitMode.GONDOLA;
-      case 7:
-        return TransitMode.FUNICULAR;
-      case 11:
-        return TransitMode.TROLLEYBUS;
-      case 12:
-        return TransitMode.MONORAIL;
-      default:
-        throw new IllegalArgumentException("unknown gtfs route type " + routeType);
-    }
+    return switch (routeType) {
+      case 0 -> TransitMode.TRAM;
+      case 1 -> TransitMode.SUBWAY;
+      case 2 -> TransitMode.RAIL;
+      case 3 -> TransitMode.BUS;
+      case 4 -> TransitMode.FERRY;
+      case 5 -> TransitMode.CABLE_CAR;
+      case 6 -> TransitMode.GONDOLA;
+      case 7 -> TransitMode.FUNICULAR;
+      case 11 -> TransitMode.TROLLEYBUS;
+      case 12 -> TransitMode.MONORAIL;
+      case 1700 -> TransitMode.CARPOOL;
```

Comment:
What about `1501 Communal Taxi Service`

Reply:
```
1506	Private Hire Service Vehicle	No	N/A
1507	All Taxi Services	No	N/A
```

These two would work even better in my view

Reply:
Well, they are not taxis - you cannot order them to come to pick you up. But if you feel that it should be mapped that way, I can do that.

Reply:
You could think of these as "fixed" (as in origin, destination and time) taxi line operated by an individual, but I agree that it's not a full match. What about 170X "private car pooling"? Just not 1700 and potentially creating a PR in gtfs spec for it?

Reply:
I think the taxis referred to in the spec are not taxi services as such, but referring to the vehicle operating the service. Many operators use taxis on a fixed route, where it doesn't make sense to use busses due to extremely lor ridership.

I'd agree that 170X is probably the best solution, so we don't use a value already defined for something else.